lib-rpc/README.md

```diff
@@ -0,0 +1,86 @@
+# ocamlformat-rpc-lib
```

Comment:
All this file would be better written in ocamlformat_rpc_lib.mli to take advantage of Odoc and avoid repeating code.

Reply:
done